packages/amplify-graphql-model-transformer/src/__tests__/model-transformer.test.ts

```diff
@@ -41,6 +41,48 @@ describe('ModelTransformer: ', () => {
     parse(out.schema);
   });
 
+  // should successfully transform simple Embeddable type (non-model) schema
+  it('should successfully transform simple Embeddable type (non-model) schema', async () => {
+    const validSchema = `
```

Comment:
added this test additionally to check a NonModel Model is successfully transformed or not